creating-virtual-machines/dedicated-cpu.adoc

```diff
@@ -95,6 +95,31 @@ spec:
 [...]
 ----
 
+Requesting dedicated CPU for QEMU emulator
```

Comment:
@vatsalparekh Are there any penalties or side effects that may appear while using this feature? I think it worth mentioning it if such exist. Otherwise why not always use it?

Reply:
I would look to @vladikr for this

Reply:
@enp0s3 using this feature the VMI will allocate an additional physical CPU. It's a trade-off that the user needs to be aware of. Definitely not something we should do blindly by default. However, those VMIs which require a more deterministic latency will need to consider moving unnecessary threads aways from the pCPUs that are dedicated for their vCPUs.

Reply:
@enp0s3 @vladikr so any changes suggested here?